pinball/run_pinball.py

```diff
@@ -101,10 +101,21 @@ def _create_workers(num_workers, factory, emailer):
     return threads
 
 
-def _run_worker(factory, emailer):
-    client = factory.get_client()
-    worker = Worker(client, DbStore(), emailer)
-    worker.run()
+def _run_worker(factory, emailer, store=None):
+    store = DbStore() if not store else store
```

Comment:
This would be more idiomatic as:

```python
store = store or DbStore()
```

... or ...

```python
if store is None:
    store = DbStore()
```